Source/APIClient/Request.swift

```diff
@@ -34,15 +36,22 @@ public protocol Request {
 
 public extension Request {
 
+    var contentType: String {
+        switch self {
+        default:
+            return "application/json"
+        }
+    }
+
     var urlRequest: URLRequest {
         let baseURL = Foundation.URL(string: Self.baseURL)!
-        let url = Foundation.URL(string: path, relativeTo: baseURL)!
+        let url = Foundation.URL(string: path, relativeTo: baseURL) ?? baseURL
```

Comment:
Not sure we want to fall back to just the baseURL in this case, probably makes more sense to just crash if we've specified an incorrect path. Otherwise it may be harder to debug the issue.